sass/atoms/_code.scss

```diff
@@ -14,6 +15,10 @@ code {
   }
 }
 
+.pre code {
```

Comment:
So those `pre` elements have a class of `pre` or was this a typo?

Reply:
Embarassing!